src/Prism.Core/Commands/AsyncDelegateCommand.cs

```diff
@@ -132,7 +145,23 @@ public bool CanExecute()
     /// <param name="parameter">Command Parameter</param>
     protected override async void Execute(object? parameter)
     {
-        await Execute(_getCancellationToken());
+        var cancellationToken = _getCancellationToken();
+        try
+        {
+            await Execute(cancellationToken)
+                .ConfigureAwait(false);
+        }
+        catch (TaskCanceledException) when (cancellationToken.IsCancellationRequested)
+        {
+            // Do nothing... the Task was cancelled
```

Comment:
Did we want to set the Exception callback letting the dev know it was canceled so maybe they can log it in their system or handle it some other way

Reply:
I almost feel like catching the exception here would be a weird API... I tend to think maybe providing something like this might provide a clearer experience:
```cs
new AsyncDelegateCommand(MyCommandTask)
    .OnTaskCancelled(() => { /* Do something... */ });
```

Also keep in mind that this catch is filtered by when the cancellationToken actually had requested the Cancel. I tend to think if you didn't request the cancel but have a TaskCanceledException that's something you should catch. This is more of a maybe you want to surface a Dialog letting someone know the request timed out.